src/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model schematypens="http://www.w3.org/2001/XMLSchema" type="application/xml" href="https://github.com/usnistgov/OSCAL/releases/download/v1.1.1/oscal_complete_schema.xsd"?>
-<catalog xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="0e9e8558-4d93-4a7f-8c87-34d4d4f9e7bd">
+<catalog xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="946e5386-e7b1-48bd-9bfd-1f5bfa59b09b">
   <metadata>
     <title>Electronic Version of NIST SP 800-53 Rev 5 Controls and SP 800-53A Rev 5 Assessment Procedures</title>
-    <last-modified>2023-10-12T00:00:00.000000-04:00</last-modified>
+    <last-modified>2023-11-14T00:00:00.000000-04:00</last-modified>
     <version>5.1.1</version>
```

Comment:
Was this version supposed to include the build, or will that be a subsequent release?